src/lib/resource/sections/check-your-answers.js

```diff
@@ -19,6 +19,72 @@ module.exports = {
                             summaryStructure: [],
                             lookup: {}
                         }
+                    },
+                    options: {
```

Comment:
There's a [metadata key](https://github.com/CriminalInjuriesCompensationAuthority/q-templates-application/blob/71e488f9e123cc6b4ab78aaa07e9a73bcc0395b3/src/lib/resource/sections/applicant-affect-duration.js#L26) for adding additional info. It might be worth adding a `ui-schema` key (or some descriptive name) in there for this type of data. I'm pretty sure it gets sent back as part of the schema response 🤔

Reply:
I've just checked and it looks like the meta key isn't returned as part of the schema response for the CYA page so I think it makes sense to leave it like this, with ```options``` chosen as the property name to keep it consistent with notation in the transformer